slides/factory-bot.markdown

```diff
@@ -0,0 +1,394 @@
+---
+layout: slide
+title: Factory Bot
+---
+
+## What is Factory Bot?
+
+factory_bot is a fixtures replacement with a straightforward definition syntax, support for multiple build strategies (saved instances, unsaved instances, attribute hashes, and stubbed objects), and support for multiple factories for the same class (user, admin_user, and so on), including factory inheritance.
+
+---
+
+## Install `factory_bot_rails`
+
+The first thing we need to do is add our gem to Gemfile in `:development, :test` groups
+
+```ruby
+group :development, :test do
+  gem "factory_bot_rails"
+end
+```
+
+Then, run `bundle` to download and instal new gem
+
+```bash
+$ bundle install
+```
+
+After that we must include `FactoryBot` to test helper file. 
+
+For this we will crete new folder spec/`support`. And `factory_bot.rb` file in it.
+
+spec/support/factory_bot.rb <!-- .element: class="filename" -->
+
+```ruby
+RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
+end
+```
+
+Just the following line down the file `rails_helper.rb`
+
+spec/rails_helper.rb <!-- .element: class="filename" -->
+
+```ruby
+Dir[File.dirname(__FILE__) + '/support/*.rb'].each { |file| require file }
```

Comment:
May be you should to describe somewhere, what this code do? and why we need this